### ISSUE TITLE
[med] JAWS+Chrome: Create New Group: User is not notified when maximum participants selected

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5669]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has 10+ contacts or recent chats that display in the new group panel.

1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Tab over to the new chat floating action icon button ( + ) in the left side panel and press Enter key.
3. Tab to the 'New Group' option in the flyout menu and press Enter key.
4. Add 7 contacts to the group chat by tabbing over to a contact and pressing the Enter key.
5. Add 1 more contact, note what is displayed and announced.

## Expected Result:
The user should be notified when the maximum number of participants are selected and all other contacts are removed from the panel. i.e. "You've reached the maximum number of participants for a group chat."

## Actual Result:
The focus is moved to the search field when the limit is reached (8th participant selected) and the warning text below is never announced. The remaining contacts are removed from the panel and only the 8 selected contacts remain.

HTML
`<div class="css-1dbjc4n" style="padding-bottom: 20px; padding-right: 20px; padding-left: 20px;"><div dir="auto" class="css-901oao" style="color: rgb(125, 139, 143); font-family: GTAmericaExp-Regular; font-size: 13px; line-height: 18px;">You've reached the maximum number of participants for a group chat.</div></div>`

## Workaround:
No workaround

## Area issue was found in:
Create new group overlay

## Failed WCAG checkpoints
4.1.3

## User impact:
The screen reader users will not know the maximum number of participants have been reached unless they navigate to the text. They will be confused by the removal of the rest of the available contacts.

## Suggested resolution:
Use aria-live="polite" attribute with the text container so changes are announced.
https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5186890_Chrome_Jaws-Create_New_Group-User_not_notified_when_maximum_participants_selected](https://user-images.githubusercontent.com/88733897/136172745-c4cdf77c-2b30-445c-be27-1019256efb82.png)

https://user-images.githubusercontent.com/88733897/136172757-c23dcee1-3af1-4da0-8263-c4d5a9291715.mp4

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
